flogger/middleware/src/main/java/io/spine/logging/flogger/backend/Platform.java

```diff
@@ -235,22 +236,47 @@ protected ContextDataProvider getContextDataProviderImpl() {
   }
 
   /**
-   * Returns whether the given logger should have logging forced at the specified level. When
-   * logging is forced for a log statement it will be emitted regardless or the normal log level
-   * configuration of the logger and ignoring any rate limiting or other filtering.
-   * <p>
-   * This method is intended to be invoked unconditionally from a fluent logger's
+   * Returns whether the given logger should have logging forced at the specified level.
+   *
+   * <p>When logging is forced for a log statement, it will be emitted regardless.
+   * Or, the normal log level configuration of the logger and ignoring any rate limiting or
+   * other filtering.
+   *
+   * <p>This method is intended to be invoked unconditionally from a fluent logger's
    * {@code at(Level)} method to permit overriding of default logging behavior.
    *
    * @param loggerName the fully qualified logger name (e.g. "com.example.SomeClass")
    * @param level the level of the log statement being invoked
-   * @param isEnabled whether the logger is enabled at the given level (i.e. the result of calling
+   * @param isEnabled whether the logger is enabled at the given level (i.e., the result of calling
    *     {@code isLoggable()} on the backend instance)
    */
   public static boolean shouldForceLogging(String loggerName, Level level, boolean isEnabled) {
     return getContextDataProvider().shouldForceLogging(loggerName, level, isEnabled);
   }
 
+  /**
+   * Obtains a custom logging level set to the logger with the given name via
```

Comment:
```suggestion
   * Obtains a custom logging level for the logger with the given name via
```

So that `set` is not used twice within the sentence.